src/main/java/code401challenges/linkedlist/LinkedList.java

```diff
@@ -128,17 +124,44 @@ public int size(){
     }
 
 
+
     public int kthFromTheLast(int k){
         int size = this.size();
         if(k > size){
             throw new NullPointerException("Not a valid node index, list size is " + size);
         }
         Node currNode = this.head;
-       int stepper = 1;
+       int stepper = 0;
        while(stepper < k){
            stepper++;
            currNode = currNode.next;
        }
         return currNode.value;
     }
+
+
+
+    public static LinkedList mergeLists(LinkedList one, LinkedList two){

```

Comment:
I like that you made this static 👌  That way you don't have to call it on an instance of LinkedList... just like we do with Math.random()